### DISCUSSION TITLE
[914710] Cat Quest II 本体

### DISCUSSION THREAD

**Discussion by a1750997036** [pjy612/SteamManifestCache#395] [Share]
[914710] ：Cat Quest II

keys：

```
914711;7a9034278cc1f859bc64f942f36c04f11d114e9073a512cf40c1e14b12682ea3
```
不知道有没有对，按照获取流程来的，如果不对还请多多包涵
联系TG，code
`
BE0052B045E167EB
`

[914710_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13986572/914710_config.vdf.txt)
[914711_3039696546619004998.manifest.txt](https://github.com/pjy612/SteamManifestCache/files/13996188/914711_3039696546619004998.manifest.txt)

**Comment by ZEERDEER** [comment]
格式不对 #26

**Reply by a1750997036** [reply]
感谢指正

**Reply by pjy612** [reply]
还是不对
* 标题
* 正文密钥格式

**Reply by a1750997036** [reply]
现在这个种格式可以了吗

**Reply by pjy612** [reply]
格式不对
* 标题 未包含 appid
* key 正文请独立出来
* 请附带清单

**Reply by a1750997036** [reply]
现在呢，如果可以了请联系一下我的TG，求求

**Reply by a1750997036** [reply]
这是我的code
`
7850A00D251CD103
`

请问下2.13.7在哪下载我目前的版本被禁止使用了

**Reply by ZEERDEER** [reply]
不对 #26

**Reply by pjy612** [reply]
还缺个清单

**Reply by a1750997036** [reply]
补了

**Reply by ZEERDEER** [reply]
😅
